tests/layer_tests/tensorflow_tests/test_tf_Slice.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestSlice(CommonTFLayerTest):
+    def create_slice_net(self, input_shape, input_type, begin_value, size_value):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            input_x = tf.compat.v1.placeholder(input_type, input_shape, 'input_x')
+            begin = tf.constant(begin_value, tf.int32)
+            size = tf.constant(size_value, tf.int32)
+            tf.raw_ops.Slice(input=input_x, begin=begin, size=size)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+        return tf_net, ref_net
+
+    test_data_basic = [
+        dict(input_shape=[6], input_type=tf.float32, begin_value=[2], size_value=[2]),
+        dict(input_shape=[2, 5, 3], input_type=tf.int32, begin_value=[0, 1, 0], size_value=[-1, 1, -1]),
```

Comment:
not sure I understand why you introduce verification of int32, but don't introduce other possible types

Reply:
agree, maybe we can place additional types to nightly tests, not to pre-commits

Reply:
this is pre-commit. For the full coverage I think we have separate activity. Now the main idea is to cover different shapes and types by mixing not overloading the pre-commit.
Nightly tests will be in separate tickets. The current tickets are to cover the common logic for the pre-commit.